nx-libs.spec

```diff
@@ -410,7 +410,14 @@ chmod a+x my_configure;
 export SHLIBGLOBALSFLAGS="%{__global_ldflags}"
 export LOCAL_LDFLAGS="%{__global_ldflags}"
 export CDEBUGFLAGS="%{?__global_cppflags} %{?__global_cflags} %{?optflags}"
-make %{?_smp_mflags} CONFIGURE="$PWD/my_configure" PREFIX=%{_prefix} LIBDIR=%{_libdir} CDEBUGFLAGS="${CDEBUGFLAGS}" LOCAL_LDFLAGS="${LOCAL_LDFLAGS}" SHLIBGLOBALSFLAGS="${SHLIBGLOBALSFLAGS}"
+IMAKE_DEFINES=''
+FORCE_TIRPC='NO'
+%if 0%{?fedora} > 27 || 0%{?suse_version} > 1500
+FORCE_TIRPC='YES'
+%endif
+IMAKE_DEFINES="-DUseTIRPC=${FORCE_TIRPC}"
+export IMAKE_DEFINES
```

Comment:
Why export IMAKE_DEFINES???

Either export of handing it over to the main Makefile as also done in below make call. IMHO, I'd prefer if that main Makefile was aware of IMAKE_DEFINES (similar to the FONT_DEFINES variable).

Reply:
I did so at first, but didn't like that too well since it added `IMAKE_DEFINES="$(IMAKE_DEFINES)"` all over the place in the top-level `Makefile`.

It may make sense to push `IMAKE_DEFINES` down, since with that a lot of options can be changed in the `nxagent` build process without using even more pass-down variables. I could potentially even rework the `XFONT` and `XEXT` stuff and drop that, instead relying on passing `imake` defines and having the `nxagent` imake system dealing with the details.

Does that make sense?